### ISSUE TITLE
request: provide nested data for `otherFormes`

### ISSUE THREAD

**Issue by favna** [favware/graphql-pokemon#902]
### Is there an existing issue or pull request for this?

- [X] I have searched the existing issues and pull requests

### Feature description

Currently the `otherFormes` field for a `Pokemon` is an array of strings, however, this field could be a nested reference to the actual Pokémon. This is currently blocked by #900 as v8 cannot handle so much data, but once that issue is resolved this should be changed.

### Desired solution

- The `otherFormes` should be a `[Pokemon!]` type instead of a `[String!]` type
- The mappers should be updated to match

### Alternatives considered

N.A.

### Additional context

The bulk of the changes are already done in #899.
